### ISSUE TITLE
How to fine tune the pre-trained model?

### ISSUE THREAD

**Issue by cansik** [fire717/movenet.pytorch#27]
Thank you for creating this project. I am looking into fine-tuning the pretrained model and wanted to ask how I should approach this? Is it enough to just load the pretrained model in `train.py` and use my custom dataset to adapt to? Or are there more steps involved?

**Comment by fire717** [comment]
Yes u can just load pretrained model in train.py and use my custom dataset.
And the pretrained model is 17 keypoints, ur data should be the same.

**Comment by cansik** [comment]
Great, that seems to work just fine, thanks. Regarding data augmentation, I found some scripts regarding data augmentation,  but I could not find any settings or something where I can control how the data is augmented. Or is there a pre-processer script to use?

**Comment by fire717** [comment]
Um...Data control...Maybe you mean Dataloader (from image loading to label making before put into network)? It's in 
`lib/data/data_tools.py`

Or if you mean how to do data augmentation, all augmentation function is written in 
`lib/data/data_augment.py`
